Client/Frontend/DefaultBrowserOnboarding/DefaultBrowserOnboardingViewModel.swift

```diff
@@ -5,6 +5,35 @@
 import Foundation
 import Shared
 
+enum InstallType: String, Codable {
+    case fresh
+    case upgrade
```

Comment:
Where do we use the upgrade case?

Reply:
Nowhere as of right now but I do plan to use it in update flow in future -> https://jira.mozilla.com/browse/FXIOS-1586

Made a small ticket to perform necessary refactors